ogr/parsing.py

```diff
@@ -10,7 +10,7 @@ class RepoUrl:
     Class that represents repo URL.
 
     Attributes:
-        repo (str): Name of the repository.
+        repo (Optional[str]): Name of the repository. Can be None if parsing is being done.
```

Comment:
```suggestion
        repo: Name of the repository. Can be `None` if parsing is being done.
```

```suggestion
        repo (Optional[str]): Name of the repository. Can be `None` if parsing is being done.
```

ogr was the first project with the google-style docstrings, in the end we have decided to not keep the trailing types after attributes. I guess, it's up to you, if we start to slowly remove them, where we touch the code or not.